src/commands/plugin/pack.rs

```diff
@@ -131,10 +138,14 @@ pub fn cmd(
         return Ok(());
     }
 
-    create_plugin_archive(&build_root, &root_dir)
+    create_plugin_archive(&build_root, &root_dir, archive_name.map(PathBuf::as_path))
```

Comment:
Можно оставить `&PathBuf`

```suggestion
    create_plugin_archive(&build_root, &root_dir, archive_name)
```